ngraph/frontend/onnx_import/src/op/reduce.cpp

```diff
@@ -31,7 +32,60 @@ namespace ngraph
         {
             namespace
             {
-                std::shared_ptr<default_opset::Constant> get_reduction_axes(const Node& node)
+                std::shared_ptr<ngraph::Node> get_dynamic_all_axes_range(const Node& node){
```

Comment:
This function adds support for input with a dynamic rank when a reduce operation should be performed on all axis. Is there any operator that generates output with dynamic rank?